jfrog-applications/ide/jetbrains-ides/plugin-configuration.md

```diff
@@ -0,0 +1,25 @@
+# Plugin Configuration
+
+To access the plugin configuration, click on the gear icon:
+
+
+![setting-button](../../.gitbook/assets/intellij/settingButton.png)
+
+## Advanced
+
+#### Downloading External Resources Through Artifactory
+
+JFrog IntelliJ Plugin requires necessary resources for scanning your projects.
+By default, the JFrog IntelliJ Plugin downloads the necessary resources needed from https://releases.jfrog.io. If the machine that runs JFrog IntelliJ Plugin has no access to it, follow these steps to allow the tools to be downloaded through an Artifactory instance, which the machine has access to:
```

Comment:
By default, the JFrog IntelliJ Plugin downloads the necessary resources needed from https://releases.jfrog.io. If the machine that runs JFrog IntelliJ Plugin has no access to it, follow these steps to allow the tools to be downloaded through an Artifactory instance, which the machine has access to:
-->
By default, the JFrog IDEA Plugin downloads the resources it requires from https://releases.jfrog.io. If the machine running IDEA has no access to it, follow these steps to allow the resources to be downloaded through an Artifactory instance, which the machine has access to: